docs/content/troubleshooting.md

```diff
@@ -1,113 +1,119 @@
  <br/>Fix this by runing the Unity installer and selecting the appropriate option. (See [Setup and installing]({{urlRoot}}/setup-and-installing#set-up-your-machine)).
  
- **Note:** When you come to build your project, in the `Assets/Fps/Config/BuildConfiguration`, do not change the `UnityGameLogic Cloud Environment` from Linux. This will cause further build errors.
+ **Note:** When building your project do not change the `UnityGameLogic Cloud Environment` field in your `BuildConfiguration.asset` from Linux. This can cause further build errors.
 
 <br/>
 
-#### Error reported 
+#### Could not discover location for dotnet.exe
 
-_Could not discover location for dotnet.exe._
+**Cause**<br/>
+Either you don't have the .NET Core SDK (x64) installed or the directory containing the dotnet executable is not in your PATH.
 
-**Issue and workarounds**<br/>
-Ensure that you have the [.NET Core SDK (x64)(Microsoft documentation)](https://www.microsoft.com/net/download/dotnet-core/2.1) installed and that the directory containing
-the dotnet executable is added to your PATH environment variable. Restart your computer
-and Unity after installing the .NET Core SDK.
+**Fix**<br/>
+
+1. Ensure that you have the correct version of [.NET Core SDK (x64)(Microsoft documentation)](https://www.microsoft.com/net/download/dotnet-core/) installed. Our supported versions are listed on the [setup page]({{urlRoot}}/setup-and-installing#set-up-your-machine)).
+1. Ensure that the dotnet executable is added to your PATH environment variable.
+1. Restart your computer after making the above changes.
 
 <br/>
 
 ## Exceptions
 
-#### Exception thrown
-_`<name of a type>` is an IComponentData, and thus must be blittable
-(No managed object is allowed on the struct)._
+#### `<name of a type>` is an IComponentData, and thus must be blittable
 
-**Issue and workarounds**<br/>
+**Cause**<br/>
 
-* The struct you defined implements `IComponentData` and contains [non-blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) fields. Types that implement
+The struct you defined implements [`IComponentData`](https://github.com/Unity-Technologies/EntityComponentSystemSamples/blob/132f511a0f36d2bb422fc807cb3a808ea18d7df5/Documentation/content/ecs_in_detail.md#icomponentdata) and contains [non-blittable](https://docs.microsoft.com/en-us/dotnet/framework/interop/blittable-and-non-blittable-types) fields. Types that implement
 `IComponentData` may only contain blittable fields.
 
-* `bool` as well as `System.Boolean` are not blittable. Please use `Improbable.Gdk.Core.BlittableBool`, a blittable alternative
-implementation we provide, in that case.
+The most common causes of this exception are `bool` and `System.Boolean`, both of which are not blittable.
+
+**Fix**<br/>
+
+ Instead of `bool` and `System.Boolean`, please use `Improbable.Gdk.Core.BlittableBool`, a blittable alternative implementation we provide, in that case.
```

Comment:
Consider rewording this sentence slightly. The `in that case` reads a bit clunky

Reply:
fixed